core/uwsgi.c

```diff
@@ -790,6 +790,7 @@ static struct uwsgi_option uwsgi_base_options[] = {
 	{"cheaper-algo", required_argument, 0, "choose to algorithm used for adaptive process spawning", uwsgi_opt_set_str, &uwsgi.requested_cheaper_algo, UWSGI_OPT_MASTER},
 	{"cheaper-step", required_argument, 0, "number of additional processes to spawn at each overload", uwsgi_opt_set_int, &uwsgi.cheaper_step, UWSGI_OPT_MASTER | UWSGI_OPT_CHEAPER},
 	{"cheaper-overload", required_argument, 0, "increase workers after specified overload", uwsgi_opt_set_64bit, &uwsgi.cheaper_overload, UWSGI_OPT_MASTER | UWSGI_OPT_CHEAPER},
+	{"cheaper-spare2-idlecount", required_argument, 0, "number of secs to decrease workers when idle", uwsgi_opt_set_int, &uwsgi.cheaper_spare2_idlecount, UWSGI_OPT_MASTER | UWSGI_OPT_CHEAPER},
```

Comment:
How do I do about spare algorithm?
It uses `cheaper-overload` (default: 3) to wait decreasing workers.

* spare algorithm uses `cheaper-idle` too.  It's 3 by default to keep compatibility to default of `cheaper-overload`.
* don't touch spare algorithm. Add `(spare2)` in option document.